include/client/audiomanager.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <unordered_map>
+#include <boost/filesystem.hpp>
+#include <SFML/Audio.hpp>
+#include <memory>
+
+enum class SoundType {
```

Comment:
I like the enum a lot since we will be able to easily remap physical audio files to certain game events without any headache